### ISSUE TITLE
Je vous conseille de nommer les types au singulier

### ISSUE THREAD

**Issue by gwendall** [Seedockh/efrei-expo-2019#23]
C'est semantiquement plus logique, quand vous avez une mutation qui retourne User pour une entrée unique et [User] pour une liste, c'est plus lisible

https://github.com/Seedockh/efrei-expo-2019/blob/5226bbe2467280328c5c9a751fae8568cd03e682/server/src/types/Users.js#L4

**Comment by Seedockh** [comment]
Je suis d'accord ! On a commencé au pluriel donc on s'y est tenus pour la suite, mais sur le fond je suis d'accord avec toi
